lib/kore/src/main/kotlin/dev/buijs/klutter/kore/project/Environment.kt

```diff
@@ -0,0 +1,23 @@
+package dev.buijs.klutter.kore.project
+
+enum class OperatingSystem(val value: String) {
+    WINDOWS("windows"),
+    MACOS("macos")
```

Comment:
don't forget linux

Reply:
Good point, thanks! Need to test on Linux to.